docs/api/paths/diagnostic/keyspace-sync.yaml

```diff
@@ -0,0 +1,61 @@
+# Copyright 2024-Present Couchbase, Inc.
+#
+# Use of this software is governed by the Business Source License included
+# in the file licenses/BSL-Couchbase.txt.  As of the Change Date specified
+# in that file, in accordance with the Business Source License, use of this
+# software will be governed by the Apache License, Version 2.0, included in
+# the file licenses/APL2.txt.
+parameters:
+  - $ref: ../../components/parameters.yaml#/keyspace
+get:
+  summary: Run a doc body through the sync function and return sync data.
+  description: |-
+    Run a document body through the sync function and return document sync data.
+    * Sync Gateway Application Read Only
+  requestBody:
```

Comment:
I think this should include a 'parameters' section to define the supported query string parameters for this endpoint (docid, rev).